src/scss/styles.scss

```diff
@@ -14,29 +10,45 @@ kontent-smart-link-overlay {
 }
 
 kontent-smart-link-element {
+  --kontent-primary-color: #0a68f5;
+  --kontent-secondary-color: #84b3fa;
+  --kontent-highlight-color: var(--kontent-secondary-color);
+
+  min-height: 24px;
+  min-width: 24px;
+
   position: absolute;
+  pointer-events: none;
+  touch-action: none;
   border: 2px dashed var(--kontent-highlight-color);
   border-radius: 5px;
-
-  min-width: 32px;
-  min-height: 32px;
-
-  &::before {
-    content: '';
-    position: absolute;
-    top: -2px;
-    right: -2px;
-    width: 32px;
-    height: 32px;
-    border-radius: 5px;
-    background-size: 60%;
-    background-repeat: no-repeat;
-    background-position: center center;
-    background-color: var(--kontent-highlight-color);
-    background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjAiIGhlaWdodD0iMjAiIHZpZXdCb3g9IjAgMCAyMCAyMCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxyZWN0IHdpZHRoPSIzLjA0NTk0IiBoZWlnaHQ9IjUuMjIxNjkiIHRyYW5zZm9ybT0ibWF0cml4KC0wLjcwNzEwOCAwLjcwNzEwNiAtMC43MDcxMDggLTAuNzA3MTA2IDE5Ljg0NjIgMy44NjUxMSkiIGZpbGw9IndoaXRlIi8+CiAgICA8cmVjdCB4PSIxNi40NjE0IiB5PSI3LjI1IiB3aWR0aD0iMTQuMzU5NSIgaGVpZ2h0PSI1LjIyMTY5IiB0cmFuc2Zvcm09InJvdGF0ZSgxMzUgMTYuNDYxNCA3LjI1KSIgZmlsbD0id2hpdGUiLz4KICAgIDxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMC4xNTM4MDkgMTkuODY1M0w1LjA3Njg5IDE4LjYzNDZMMS4zODQ1OCAxNC45NDIzTDAuMTUzODA5IDE5Ljg2NTNaIiBmaWxsPSJ3aGl0ZSIvPgo8L3N2Zz4K);
-  }
+  z-index: 10;
```

Comment:
No, it is a solution to one highlight being over another highlight, I wanted to add some value to the non-hovered state as well so that it is visible that the hovered highlight (`z-index: 20`) is higher than the non-hovered (`z-index: 10`) just by looking at the SCSS file.

The z-indexes problem from the JIRA task is solved by rendering the elements into the parent element instead of body.

Reply:
I see, makes sense